Problem1.java

```diff
@@ -1,7 +1,62 @@
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Any problem you faced while coding this :
+import java.util.HashMap;
 
+//Time Complexity :  O(n3) for the Brute Force. method subarraySum.  O(n) for the Hashmap Methodm
+//Space Complexity : O(1) for the Brute Force. method subarraySum.   O(n) for the Hashmap Methodm
+//Did this code successfully run on Leetcode : Yes 
+//Any problem you faced while coding this : Hashmap is not clear for me.
```

Comment:
Is it clear after today's Session?
If you still have any doubts reach out with your doubts.